llvm/lib/CodeGen/MachineTraceMetrics.cpp

```diff
@@ -605,7 +599,7 @@ void MachineTraceMetrics::Ensemble::verify() const {
 #ifndef NDEBUG
   assert(BlockInfo.size() == MTM.MF->getNumBlockIDs() &&
          "Outdated BlockInfo size");
-  for (unsigned Num = 0, e = BlockInfo.size(); Num != e; ++Num) {
+  for (unsigned Num = 0; Num < BlockInfo.size(); ++Num) {
```

Comment:
The left form is [preferred](https://llvm.org/docs/CodingStandards.html#don-t-evaluate-end-every-time-through-a-loop).